src/test/ui/proc-macro/amputate-span.rs

```diff
@@ -0,0 +1,63 @@
+// aux-build:amputate-span.rs
```

Comment:
Adding `// run-rustfix` to this test does not work out of the box. It ends up rewriting the code to the bad thing that we are trying to prevent:

```rust
#[amputate_span::drop_first_token]
/* what the
hey */ async use std::process::Command;

fn main() {
    Command::new("git"); //~ ERROR [E0433]
}
```

I'm guessing that's because `rustfix` does not know `Transcription::Blanks`?

Reply:
This may be somewhat related to this FIXME from #85427 that I think stems from #44215.  Fixing *that* might serve as an argument for moving away from the "infer the span from the given item's span" strategy, and using the span of the containing mod/crate instead, as discussed in [my other comment](https://github.com/rust-lang/rust/pull/90941#issuecomment-970308479)

https://github.com/ehuss/rust/blob/2c7bc5e33c25e29058cbafefe680da8d5e9220e9/src/test/ui/resolve/use_suggestion_placement.rs#L13-L16 )